src/scripts/views/member_recruits.js

```diff
@@ -0,0 +1,41 @@
+var $ = require("jquery"),
+  _ = require("underscore"),
+  Backbone = require("backbone"),
+  Template = require("../templates/member_recruits.html"),
+  config = require("../config");
+var Marionette = require("backbone.marionette");
+
+  
+  module.exports = Marionette.ItemView.extend({
+      template: Template,
+      initialize: function(options) {
+          options = options || {};
+          this.recruits = options.collection || false;
+      },
+      serializeData: function () {
+          var items = [];
+          var rec_count = 0;
+          _.each( this.recruits.toJSON(), function( recruit ) {
```

Comment:
Backbone lets you use most underscore methods directly on models & collections, so this could be `this.recruits.each(function(recruit) {` -- note that you'd then need to do `recruit.get('member').rank` since `recruit` is a model.